.github/workflows/verify.yml

```
@@ -0,0 +1,74 @@
+name: Verify Submodule Pull Request
+
+on:
+  workflow_call:
+    inputs:
+      repo:
+        required: false
+        type: string
+        default: Lombiq/Open-Source-Orchard-Core-Extensions
+
+env:
+  PULL_REQUEST_TITLE: ${{ github.event.pull_request.title }}
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  parent-pull-request-exists:
+    runs-on: ubuntu-latest
+    name: Parent PR Exists
+    steps:
+      - name: Print PR Title
+        run: echo $Env:PULL_REQUEST_TITLE
+
+      - name: Add Scripts
+        run: |
+          $code = @'
+          <#
+          .Synopsis
+          Fails the current step with the provided message. Same as "core.setFailed()" from the official toolkit
+          (https://github.com/actions/toolkit).
+          #>
+          function Set-Failed([Parameter(Mandatory=$True)] [string] $Message)
+          {
+              echo "::error::$Message"
+              exit 1
+          }
+
+          <#
+          .Synopsis
+          Returns $True if any item is coming from the pipe.
+          #>
+          function Any-Object([Parameter(Mandatory=$True, ValueFromPipeline = $True)] $Input)
+          {
+              foreach($item in $Input)
+              {
+                  return $True
+              }
+
+              return $False
+          }
+          '@
+
+          $ProfileDirectory = [System.IO.Path]::GetDirectoryName($Profile)
```

I couldn't find a definitive guideline, but based on [this longish example](https://docs.microsoft.com/en-us/powershell/scripting/developer/module/how-to-write-a-powershell-script-module?view=powershell-7.2#show-calendar-code-example) and your own convention below I would say that we should use camel-cased variable names everywhere:
```suggestion
          $profileDirectory = [System.IO.Path]::GetDirectoryName($Profile)
```